### ISSUE TITLE
Stopwatch does not work if MonitoringProxy hasn't been initialized yet.

### ISSUE THREAD

**Issue by NicolasFelix** [javamelody/javamelody#876]
Hi,
thanks again for this great project,
and I first apologise for both my English level and if I misunderstood this feature.

So, I plugged on a portion of code the **Stopwatch** class.
This works correctly on my project.

One of my coworkers then updates his dependencies to get my/those new metrics,
but it doesn't work as expected on his product (no available data into the _Statistisques Services_ chapter/node).

After some research, we found this behaviour was coming from **SERVICES_COUNTER#used** property not initialised in his context.
On my project, I previously called **MonitoringProxy.createProxy**, which creates a **MonitoringProxy** and also marks **SERVICES_COUNTER** as **used**.

Is it the expected behaviour, or did I miss something ?
_This is a low-priority issue for us, as we can get around that bug._

Let me know if you have any suggestion.

Regards,
Nicolas

**Comment by evernat** [comment]
Hi,
Merci. Et tu es tout excusé pour ton anglais, le mien n'est pas mieux :smiley:

You are right. This is a bug. Something like the following from MonitoringProxy should be called in StopWatch:
```
private static final boolean COUNTER_HIDDEN = Parameters
			.isCounterHidden(SERVICES_COUNTER.getName());
...
SERVICES_COUNTER.setDisplayed(!COUNTER_HIDDEN);
SERVICES_COUNTER.setUsed(true);
```

**Comment by NicolasFelix** [comment]
Merci/thanks @evernat, good job 👍

**Comment by evernat** [comment]
Fixed by a7bc53b and ready for the next release (1.81).
A snapshot build is available at:
http://javamelody.org/downloads/javamelody-core-1.81.0-SNAPSHOT.jar

**Comment by NicolasFelix** [comment]
Great.
Merci encore @evernat 👍
